Typography/src/actions/Actions.styles.tsx

```diff
@@ -0,0 +1,27 @@
+import { css, styled } from '@stitches/react';
+import BaseTypography from '../base-typography/BaseTypography';
+
+export const StyledActions = styled(BaseTypography);
+export const actionsStyles = css({
+  fontFamily: '$fonts$actions',
+  fontWeight: '$fontWeights$bold',
+  letterSpacing: '6%', // TODO: REVIEW
+
+  variants: {
+    variant: {
+      small: {
+        fontSize: '$fontSizes$2',
```

Comment:
Out of curiosity where do we stand on shorthand notation for this? Should I avoid using just $2 and always explicitly use $fontSizes$2?

Reply:
I forgot about that part honestly. Too much reflex from other theming systems. Also, the first couple styles i applied didn't work with the direct mapping. I'll go back and look at these and see what I can shorten. Potentially lots. But your question is good.... fontSize: $2 probably reads pretty well over what I have. In other places I feel like maybe the mapping is not as inherent?

Reply:
Agreed, I think we can see what feels right as the developer. If it feels like this is cryptic for the specific use case then we put the full key, otherwise $2 does read quite well for something like fontSize.

Reply:
I'm retracting my comment, now that I have updated it everywhere, it feels better in all the instances I had here.

Reply:
383e3aa